srcpkgs/interception-tools/files/udevmon/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+if [ -r ./conf ]; then
+	. ./conf
+	if [ -n "${CONF_FILE}" ]; then
+		OPTS="${OPTS} -c '${CONF_FILE}'"
+	fi
+fi
+
+exec 2>&1
```

Comment:
Nit: you want this to happen before sourcing `.conf`, to catch any possible errors.

Reply:
Nice one. I'll change it.